client/src/components/splitList.jsx

```diff
@@ -67,9 +84,16 @@ const SplitList = (props) => {
   return (
   <ul>
     {props.guests.map((guest, i) =>
-      <IndividualOwes key = {i} firstName = {guest.guestName} paymentOwed = {paymentsOwed[guest.guestName]} />
+      <IndividualOwes key = {i} guestName = {guest.guestName} paymentOwed = {paymentsOwed[guest.guestName]} />
     )}
-    Total: ${billWithTip.toFixed(2)}
+    <div>
+      <hr />
+      </div> Preliminary Total: ${billTotalWithoutTipOrTax.toFixed(2)} <div>
+      <div> Tip Amount: ${tipAmount.toFixed(2)} </div>
+      <div> Tax: ${taxAmount.toFixed(2)} </div>
+      <div> Final Total: ${billWithTipAndTax.toFixed(2)} </div>
```

Comment:
For the future, we should probably switch these over from divs to something more appropriate, though it may not be a biggie.

Reply:
I will deal with that next week during CSS fun times